playbook/buildfarmer/buildfarmer_triage_tools.md

```diff
@@ -26,7 +23,7 @@ Check the [testdb usage](#testdb-usage) section for more information on how to u
 
 The buildfarmer repository stores a local database with information from test regressions, jobs, known issues, etc. The database scripts provide a way to check more information based on the data stored in the database. These scripts are located in the [database/scripts](../../database/scripts/) folder.
 
-*Important note*: Currently, the database is updated manually using [databaseFetcher.py](../../database/databaseFetcher.py), so the database scripts will only provide consistent results if the database is up to date. To update the database, install the dependencies using `python3 -m pip install -r requirements.txt` in the root folder and then `python3 databaseFetcher.py` or `./databaseFetcher.py`
+*Important note*: Currently, the database is updated daily, so the database scripts will only provide consistent results if the database is up to date.
```

Comment:
I think we should be able to query the database and get the last time it was updated, is that something we have today?

Reply:
That's a good point. 

From https://stackoverflow.com/a/15915513:

> SQLite does not track modification or creation time on it's own. You must add this functionality yourself.

I think we could figure out a way to add metadata to buildfarmer database

Reply:
However, we do have a `last_time_updated` table that contains the last build number of each build

Reply:
Also, you can check the last commit time in Github, and that would give the answer

Reply:
Added a note here: fa95953